opentelemetry-stdout/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "opentelemetry-stdout"
+version = "0.1.0"
+description = "An OpenTelemetry exporter for stdout"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-stdout"
+repository = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-stdout"
+readme = "README.md"
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
+    "asynchronous",
+]
+keywords = ["opentelemetry", "tracing", "metrics"]
+license = "Apache-2.0"
+edition = "2021"
+rust-version = "1.60"
+
+[features]
+trace = ["opentelemetry_api/trace", "opentelemetry_sdk/trace", "futures-util"]
```

Comment:
Should we have all features enabled by default since this crate is pramarily used for testing/examples?

Reply:
Possibly, this matches the other exporters where you specify the signals you want to export, but I'm open to changing it in the future.